### PR TITLE
 build-fail-reminder: don't use cgi.escape, but rather html escape the fixed string directly

### PR DESCRIPTION
**Pull request by DimStar77**

### CODE REVIEW

File: build-fail-reminder.py [hunk 1]
```diff
@@ -188,7 +188,7 @@ def main(args):
                 # Package failed to build for 6 weeks - file a delete request
                 r = osc.core.Request()
                 r.add_action('delete', tgt_project=project, tgt_package=package)
-                r.description = cgi.escape("[botdel] Package has failed to build for >= 6 weeks")
+                r.description = escape("[botdel] Package has failed to build for >= 6 weeks")
```
Comment (coolo) [inline build-fail-reminder.py:191]: just using `&gt;= 6 weeks` is out of style? :)

Comment (DimStar77) [reply build-fail-reminder.py:191]: sure; is an option - I don't mind changing it and not using escape (was just a 1 for 1 replacement without brain muscle)

**Comment by DimStar77** [comment]
after we merge / deploy this, we should set the counter of everything that is >= 6 to 5 - to make sure the botdel action fires (in the cache file that is, of course)

**Review by coolo — APPROVED** [review]

**Comment by coolo** [comment]
So tell me about that cache file

**Pull request merged**